### ISSUE TITLE
Loading a point map from JSON without SV causes upload to fail

### ISSUE THREAD

**Issue by Koenvh1** [RuurdBijlsma/locationestimatr2#4]
When importing a point map from a JSON file, the points are loaded by the Streetview view stays black. When immediately uploading the map afterwards (so without clicking on the map or anything), it will fail due to the image from Streetview not being there.

**Comment by RuurdBijlsma** [comment]
Do you mean when loading points from json that have been changed by hand in the json file?

**Comment by Koenvh1** [comment]
I mean this. There are points, but no Streetview image. When I tried to upload this (by clicking import, selecting the JSON, entering a name and clicking upload), it failed (well, kept "loading" infinitely). As soon as I added another point manually (by clicking on the map), it submitted within a second or so.

![image](https://user-images.githubusercontent.com/5168825/83081671-7efda980-a081-11ea-853a-d9814d45eb35.png)

The JSON file itself (as ZIP, because GitHub does not allow me to upload a JSON file directly):
[Snelwegen.zip](https://github.com/RuurdBijlsma/locationestimatr2/files/4692363/Snelwegen.zip)
Though I believe it works with any JSON points file (although I have not tested that yet).
